packages/sui-i18n/src/slugify.js

```diff
@@ -11,7 +11,7 @@ export function slugify(str) {
   return str
     .toLowerCase()
     .replace(/.{1}/g, replaceCharIfNeeded)
-    .replace(/[^a-z0-9 -]/g, '') // remove invalid chars
+    .replace(/[^a-z0-9 - .]/g, '') // remove invalid chars only letters numbers, dashes and dots
```

Comment:
What about not parching and using what @davecarter has just done [here](https://github.com/SUI-Components/sui/pull/363/files) ?
Our slugs will indeed be more consistent.

Reply:
I wont want to change the tool drastically so if we do that I'll increase the sui i18n one major in order to be pretty sure to avoid side effects

Reply:
@davidbarna @davecarter decided to close his tool so... this is the only place with an slufigy